modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/plugin/PluginExtendedDeploymentProperties.java

```diff
@@ -20,26 +21,38 @@
  */
 public class PluginExtendedDeploymentProperties extends ExtendedDeploymentProperties {
 
-  private final DeployableArtifactDescriptor deployableArtifactDescriptor;
+  private final BundleDescriptor pluginBundleDescriptor;
+  private final ArtifactDescriptor deployableArtifactDescriptor;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "deployableArtifactDescriptor" transient or serializable. ([squid:S1948](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1948))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7687&issues=AWj0M4hHotIXnmsUdjY5&open=AWj0M4hHotIXnmsUdjY5)

[ISSUE_KEY]: AWj0M4hHotIXnmsUdjY5